### DISCUSSION TITLE
Adding a PNG image without background

### DISCUSSION THREAD

**Discussion by Grefuwan** [TomSchimansky/CustomTkinter#1649] [General]
I am trying to add a PNG image of a bug as the [documentation](https://customtkinter.tomschimansky.com/documentation/utility-classes/image) says, but it gets all the time with background no matter if I set it as "dark_image" or "light_image". 

![image](https://github.com/TomSchimansky/CustomTkinter/assets/46687935/d484cf9f-554d-4050-85ed-6092db015d09)

```import customtkinter
from PIL import Image

pathC0 = "<Path to the Image>"

customtkinter.set_appearance_mode("system")
customtkinter.set_default_color_theme("dark-blue")

root = customtkinter.CTk()
root.geometry("1020x680")

imgCuca0 = customtkinter.CTkImage(dark_image=Image.open(pathC0), size=(150,150))

frame = customtkinter.CTkFrame(master=root)
frame.pack(pady=20, padx=60, fill="both", expand=True)

imgC0Label = customtkinter.CTkLabel(master=frame, image=imgCuca0, text="")
imgC0Label.pack(pady=12, padx=10)

root.mainloop()

**Comment by Akascape** [comment]
@Grefuwan Try changing the png image.

**Comment by TomSchimansky** [comment]
I would say there's something wrong with the image.
